doc/REAMDE.md

```diff
@@ -0,0 +1,16 @@
+# Openfisca-Core reference documentation
+
+Source of the [openfisca-core reference documentation](http://openfisca.readthedocs.io/).
+
+## Installing
+
+```sh
+    pip install -r requirements.txt
```

Comment:
> Either indent or use the triple-backtick syntax to trigger code blocks. I recommend triple-backticks over indentation, since it allows specifying the language.